fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/FedoraTypes.java

```diff
@@ -99,6 +99,8 @@
 
     String FCR_TOMBSTONE = "fcr:tombstone";
 
+    String FCR_TX = "fcr:tx";
```

Comment:
This is already defined in [TransactionConstants](https://github.com/fcrepo4/fcrepo4/blob/main/fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/session/TransactionConstants.java#L42)... although FedoraTypes seems like a logical home. Would you be interested in refactoring these into a single constant?

Reply:
@awoods I've modified `TX_PREFIX` to be defined in terms of the `FCR_TX` constant.